### DISCUSSION TITLE
@react-spring peer dependency issue

### DISCUSSION THREAD

**Discussion by ferrarisMat** [pmndrs/drei#678] [Q&A] [answered]
- `three` version: 0.135.0
- `@react-three/fiber` version: 5.3.10
- `@react-three/drei` version: 7.27.1
- `node` version: 16.4.1
- `npm` (or `yarn`) version: 1.22.17

### Problem description:

Peer dependency issue since update to `7.27.1`

```
./node_modules/@react-spring/three/dist/react-spring-three.esm.js
Module not found: Can't resolve '@react-three/fiber' in '/Users/user/Desktop/perso/website/node_modules/@react-spring/three/dist'
```

I tested to upgrade all my dependencies without success

**Comment by joshuaellis** [comment, marked as answer]
you're using a very outdated version of `@react-three/fiber`considering that version of drei expects `@react-three/fiber@>=6`

**Reply by ferrarisMat** [reply]
Oh, I haven't seen this detail, thank you ! 💯

**Comment by yulluone** [comment]
I am getting this error 

`./node_modules/@react-spring/three/dist/react-spring-three.esm.js:6:0 Module not found: Can't resolve 'three'
`
when trying to 
`import { Canvas } from "@react-three/fiber";
import { OrbitControls, Preload, useGLTF } from "@react-three/drei";`

I have these dependencies installed

- "@react-three/drei": "^9.68.6",
- "@react-three/fiber": "^8.13.0",

**Reply by MJunaid648** [reply]
same issue
